### PR TITLE
Fixes resisting out of cryo not working 

### LINKED ISSUE #15401: Resisting out of cryo cell is broken
If you move while conscious inside cryo cell you start resisting out of it, but it never works. I tested it and apparently it's using addtimer that breaks the thing.
https://github.com/tgstation/-tg-station/blob/e554077458904323d3bd66cd4be37fd852ec41a6/code/modules/atmospherics/machinery/components/unary_devices/cryo.dm#L146
@neersighted 

edit: apparently it only fails when resisting via movement, the resist button works fine.

### PR DESCRIPTION
**Pull request by phil235**
...when you use moving instead of the resist button.

Also changes addtimer into a do_after().

Fixes #15401

### CODE REVIEW

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm [hunk 1]
```diff
@@ -141,13 +141,11 @@
 		return occupant
 
 /obj/machinery/atmospherics/components/unary/cryo_cell/container_resist(mob/user)
-	usr << "<span class='notice'>You struggle inside the cryotube, kicking the release with your foot... (This will take around 30 seconds and you need to stand still.)</span>"
+	user << "<span class='notice'>You struggle inside the cryotube, kicking the release with your foot... (This will take around 30 seconds and you need to stand still.)</span>"
```
Comment (neersighted) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: No need to say anything about moving because you're in a container...

Comment (phil235) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: But how does the player know that the resist process will free him in 30 seconds if we don't tell him?

Comment (duncathan) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: Just say "this will take 30 seconds" without "you need to stand still"

Comment (neersighted) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: Yeah, thats what I meant.

Comment (phil235) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: oh, ok, sure.

**Pull request merged**